packages/tasks/src/tasks/image-text-to-text/data.ts

```diff
@@ -43,8 +43,8 @@ const taskData: TaskDataCustom = {
 	metrics: [],
 	models: [
 		{
-			description: "Cutting-edge vision language model that can also localize texts in images.",
-			id: "liuhaotian/llava-v1.6-34b",
+			description: "Cutting-edge vision language model that can take multiple image inputs.",
+			id: "facebook/chameleon-7b",
```

Comment:
Probably also useful to add https://huggingface.co/llava-hf/llava-v1.6-mistral-7b-hf

Reply:
I think it's a bit outdated now with new interleaved/video-image llava models, but wasn't sure where to fit them in since there's going to be `video-text-to-text`